### PR TITLE
[WFLY-10912] - CodecSessionConfig#findSessionId() causes an incorrect…

### PR DESCRIPTION
**Pull request by adamecp**
CodecSessionConfig#findSessionId() causes an incorrect JSESSIONID Set-Cookie header
JIra issue: [https://issues.redhat.com/browse/WFLY-10912](https://issues.redhat.com/browse/WFLY-10912)

### CODE REVIEW

**Comment by pferraro** [comment]
@adamecp Did you intend to submit a fix for this issue as well?  I only see a test case reproducing the problem.

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/response/CodecSessionConfigTestCase.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2020, Red Hat Inc., and individual contributors as indicated
```
Comment (bstansberry) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/response/CodecSessionConfigTestCase.java:3]: 2022?

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/response/CodecSessionConfigTestCase.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    public void testResponseSessionId() throws Exception {
+        HttpGet httpget = null;
+        HttpResponse response = null;
+        String stringUrl = url.toString() + URL_PATTERN+"index.html";
+        httpget = new HttpGet(stringUrl);
+        httpget.addHeader("Cookie", "JSESSIONID=foobar");
+        response = this.httpclient.execute(httpget);
+        Header[] headers = response.getHeaders("Set-Cookie");
+        if(headers.length > 0) {
+            Assert.fail("The expected behavior is that WildFly/EAP 7 should respond without JSESSIONID Set-Cookie header. See https://issues.redhat.com/browse/WFLY-10912");
```
Comment (bstansberry) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/response/CodecSessionConfigTestCase.java:86]: Please remove the "EAP 7" here and in the javadoc. Thanks!

**Review by rhusar — CHANGES_REQUESTED** [review]
For this pull request to be mergeable, the test would need to be commited as `@Ignored`.

**Comment by pferraro** [comment]
We can close this - it was already addressed in full by https://github.com/wildfly/wildfly/pull/16019

**Pull request closed without merging**